### PR TITLE
data: project addition by kunal kashyap #682

### PR DESCRIPTION
**Pull request by kunalkashyap-1**
## Description

Added new project to the project.json file.
closes issue #682 

## Screenshots
image

### CODE REVIEW

**Comment by priyankarpal** [comment]
could you add issue number to the description?

**Comment by kunalkashyap-1** [comment]
Added the issue number 😁

File: src/DB/projects.json [hunk 1]
```diff
@@ -1211,6 +1211,24 @@
...
+    "Instagram": "",
+    "YouTube": "",
+    "Twitter": "https://twitter.com/kunal_code"
+  },
+  "Projects": [
+    {
+      "link": "https://feedparsify.netlify.com/",
+      "title": "News API",
+      "description": "This a news API/Rss feed parser which scrapes news from websites and provides them in simple json format for the ease of developers. Read the docs and use the console page to live test the API.",
+      "tech": "[React, Material UI, Express js, cherrio]"
```
Comment (priyankarpal) [inline src/DB/projects.json:1228]: change this 
``` suggestion
      "tech": ["React", "MUI", "Express js", "cherrio"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by kunalkashyap-1** [comment]
Fixed that error from my side, do tell me if it needs any other fix

File: src/DB/projects.json [hunk 2]
```diff
@@ -1211,6 +1211,24 @@
...
+    "Instagram": "",
+    "YouTube": "",
+    "Twitter": "https://twitter.com/kunal_code"
+  },
+  "Projects": [
+    {
+      "link": "https://feedparsify.netlify.com/",
+      "title": "News API",
+      "description": "This a news API/Rss feed parser which scrapes news from websites and provides them in simple json format for the ease of developers. Read the docs and use the console page to live test the API.",
+      "tech": ["React", "Material UI", "Express js", "cherrio"],
```
Comment (priyankarpal) [inline src/DB/projects.json:1228]: remove `,` 
```suggestion
      "tech": ["React", "Material UI", "Express js", "cherrio"]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by kunalkashyap-1** [comment]
done 😅

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**